### DISCUSSION TITLE
Add card view to `list` shortcode

### DISCUSSION THREAD

**Discussion by Farbklex** [nunocoracao/blowfish#795] [Ideas]
The [list](https://blowfish.page/docs/shortcodes/#list) shortcode shows recent articles which can be filtered by type.

On the home page, the normal list, tags and taxonomie pages, blowfish already allows showing content either as a list or via a card view.

It would be nice if we could also implement the same cardview to the `list` shortcode.

**Comment by nunocoracao** [comment]
Thanks for the suggestion @Farbklex. Please open a new feature request for this one. I like the idea

**Comment by knowinginner** [comment]
I would really like this feature too. Faust

**Comment by nunocoracao** [comment]
@Farbklex if you opened the ticket can you share here? That way @knowinginner can vote and contribute there

**Reply by knowinginner** [reply]
Thanks dear nunocoracao for developing this theme. Currently I am using wowchemy academic and adding card list to short code will open portfolio opportunity. 

Faust

**Comment by nunocoracao** [comment]
@Farbklex @knowinginner added in https://github.com/nunocoracao/blowfish/pull/1082

Will be available in the next release
